solutions/gold/usaco-863.mdx

```diff
@@ -7,6 +7,21 @@ author: Sofia Yang, Ryan Chou
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_teamwork_gold_dec18.html)
 
+
+<Spoiler title="Hint 1">
+
+Notice the constraints on the input, $\mathcal{O}(N \cdot K)$ is feasible.
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+We notice that we just need to find the best possible sums for the first $i$ cows as we iterate $i$ from $1$ to $N$.
```

Comment:
is this supposed to be a hint at dp? im ngl for more inexperienced users this hint probably means nothing